lmdb-sys/build.rs

```diff
@@ -21,6 +24,7 @@ fn main() {
             .file(lmdb.join("midl.c"))
             // https://github.com/LMDB/lmdb/blob/LMDB_0.9.21/libraries/liblmdb/Makefile#L25
             .opt_level(2)
+            .flag(&format!("-include{}", undefine_have_memalign_h.to_str().unwrap()))
```

Comment:
Note that -include is not recognized by MSVC.

Reply:
Ah, right, I've pushed an update (1ce4bc6) that doesn't set the flag if `build.get_compiler().is_like_msvc()`.